server/chat-plugins/scavengers.js

```diff
@@ -33,6 +34,13 @@ const HISTORY_PERIOD = 6; // months
 
 const ScavengerGames = require("./scavenger-games");
 
+let scavengersData = {recycledHunts: []};
+try {
+	scavengersData = JSON.parse(FS(DATABASE_FILE).readIfExistsSync());
+} catch (err) {
+	console.log(err);
+}
```

Comment:
Please don't abuse `try`/`catch` like this. I was thinking more along the lines of this.

```javascript
const contents = FS(DATABASE_FILE).readIfExistsSync();
const scavengersData = contents ? JSON.parse(contents) : {recycledHunts: []};
```

Reply:
I believe this would cause Scavengers to crash if the file can't be accessed for reasons other than it not existing. Is that alright?

Reply:
Yes, and that's a good thing. If the database file couldn't be loaded, chances are things would break later on, and data would end up being lost.